plugins/BEdita/Core/src/Model/Entity/Profile.php

```diff
@@ -41,4 +41,15 @@
  */
 class Profile extends ObjectEntity
 {
+
+    /**
+     * {@inheritDoc}
+     */
+    protected $_hidden = [
```

Comment:
They are. This piece of code is completely useless. 🙃  I'm reverting these changes.